### ISSUE TITLE
No spreadoperator needed on NodeLists

### ISSUE THREAD

**Issue by Napanee** [schorfES/pacto#9]
It's not necessary, to use the spreadoperator on a NodeList (result of `querySelectorAll`).
You can use `foreach` directly on `elements`.

https://github.com/schorfES/pacto/blob/4108696ebd9716f200e54037226a2e2f3ba4ac7f/src/actions/Initialize.js#L55

https://github.com/schorfES/pacto/blob/22284775d186a0f64c9390fb17d1a4058842659f/src/actions/InitializeLazy.js#L71

To support older Browser like the IE, use

```
Array.prototype.forEach.call(elements, (element) => {
	// ...
});
```

**Comment by Napanee** [comment]
To support older Browser like the IE, use

```
Array.prototype.forEach.call(elements, (element) => {
	// ...
});
```

**Comment by schorfES** [comment]
Thank you, pacto expects a support of `NodeList.prototype.forEach`. I've added an [hint](https://github.com/schorfES/pacto#support-nodelistprototypeforeach) to the docs for older browsers. It's available in version 0.4.0
